app/src/mixins/update-user-mixin.ts

```diff
@@ -39,7 +39,9 @@ export class UpdateUserMixin extends Vue {
         name: orgInfo.name
       }
 
-      return UpdateLdUser(userContext, orgContext)
+      if (userContext || orgContext) return UpdateLdUser(userContext, orgContext)
```

Comment:
Without this, if the user wasn't logged in, UpdateLdUser() would show a console error.